NewForce-Cohort-5/tabloidmvc-the-spaghetti-squad
# Ticket #9 - View All Categories

# Description

For this ticket, it is acceptable to treat all users as admin users. As an admin, you will see a list of all the available categories, which will be displayed in ascending order (alphabetically) by name. The edit, details, and delete links for each category were commented out as there is no functionality for them and will be implemented for their respective tickets.


## Type of change

# Testing Instructions
1) Run the command "git fetch --all" to pull down this new branch locally
2) Run the command "git checkout cr-ticket9" to switch over to this branch for viewing/testing
3) Go ahead and log in as an admin with the email admin@example.com
4) To make sure the list of all categories displaying in the browser is correct, you can run this SQL query statement to validate results: 

SELECT  *
FROM Category;

4) Navigate over to the "Category Management" link in the navbar to view a list of all categories that are displayed in ascending order (alphabetically)
5) Below should be what is displayed on your browser for the list of categories:

ID     Name
14	Camping
2	Close Magic
10	Cooking
6	Cthulhu Sightings
9	Entertainment
7	History
8	Home and Garden
5	Improv
12	Movies
11	Music
3	Politics
13	Regrets
4	Science
1	Technology

# Checklist:
